api/src/main/java/org/eclipse/microprofile/config/spi/ConfigSource.java

```diff
@@ -83,63 +84,76 @@
      * @return the map containing the properties in this config source
      */
     default Map<String, String> getProperties() {
-        Map<String, String> props = new HashMap<>();
+        final Map<String, String> props = new HashMap<>();
         getPropertyNames().stream().forEach((prop) -> props.put(prop, getValue(prop)));
         return props;
     }
 
     /**
-     * Gets all property names known to this config source, potentially without evaluating the values.
+     * Gets all property names known to this config source, potentially without
+     * evaluating the values.
      *
-     * For backwards compatibility, there is a default implementation that just returns the keys of {@code getProperties()}
-     * slower ConfigSource implementations should replace this with a more performant implementation
+     * For backwards compatibility, there is a default implementation that just returns
```

Comment:
Javadoc improvements: normally every paragraph after the first sentence should be wrapped in `<p>` tags.